test/Algebra/Graph/Test/GraphRules.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE TemplateHaskell #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module     : Algebra.Graph.Test.Graph
+-- Copyright  : (c) Andrey Mokhov 2016-2018
+-- License    : MIT (see the file LICENSE)
+-- Maintainer : andrey.mokhov@gmail.com
+-- Stability  : experimental
+--
+-- Testsuite for "Algebra.Graph" rewrite-rules
+-----------------------------------------------------------------------------
+module Algebra.Graph.Test.GraphRules where
+
+import Data.Maybe (fromMaybe)
+
+import Algebra.Graph
+import Algebra.Graph.Internal
+
+import qualified Test.Inspection as I
```

Comment:
Let's import non-qualified to reduce some `I.` noise?

Reply:
I wanted to avoid the clash with `(===)` from `Algebra.Graph`, but hiding it is simpler :)